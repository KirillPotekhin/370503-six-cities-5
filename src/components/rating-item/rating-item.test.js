import React from "react";
import renderer from "react-test-renderer";
import RatingItem from "./rating-item";

const noop = () => {};

const ratingTitle = `perfect`;

it(`Should RatingItem render correctly`, () => {
  const tree = renderer
    .create(<RatingItem
      rating={`3`}
      value={5}
      ratingTitle={ratingTitle}
      onRatingChange={noop}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
