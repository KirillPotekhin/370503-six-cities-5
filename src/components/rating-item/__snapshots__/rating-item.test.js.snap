// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingItem render correctly 1`] = `
Array [
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="5-stars"
    onChange={[Function]}
    required={true}
    type="radio"
    value={5}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="5-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
